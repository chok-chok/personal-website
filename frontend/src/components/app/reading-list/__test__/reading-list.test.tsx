import * as React from 'react';
import ReadingList from '../reading-list';
import { shallow } from 'enzyme';

describe('Test ReadingList component', () => {
  it('should renders correctly', () => {
    const tree = shallow(<ReadingList />);
    expect(tree).toMatchSnapshot();
  });
});
