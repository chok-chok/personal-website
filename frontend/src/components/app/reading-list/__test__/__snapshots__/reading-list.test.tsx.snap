// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ReadingList component should renders correctly 1`] = `
<Fragment>
  <styled__ReadlingListHeaderSection>
    <ReadingListHeader
      bodyText="I love to read about various of topics. here are some selection of books which Iâ€™d like to share my rating and reviews"
      headerText="Reading List"
    />
  </styled__ReadlingListHeaderSection>
  <styled__ActiveReadsSection>
    <styled__ActiveReadsInnerWrapper />
  </styled__ActiveReadsSection>
  <styled__ReadsHistorySection />
</Fragment>
`;
